test/Microsoft.AspNet.Server.KestrelTests/KestrelServerInformationTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNet.Server.Kestrel;
+using Microsoft.Extensions.Configuration;
+using Xunit;
+
+namespace Microsoft.AspNet.Server.KestrelTests
+{
+    public class KestrelServerInformationTests
```

Comment:
While we're adding tests, can we add a test to verify that when you try to call `KestrelServer.Start` with `ThreadCount==0`, it throws?